linera-execution/tests/fee_consumption.rs

```diff
@@ -74,15 +97,21 @@ async fn test_fee_consumption(spends: Vec<FeeSpend>) -> anyhow::Result<()> {
             sum.saturating_add(spent_fees)
         });
 
+    let authenticated_signer = if owner_balance.is_some() {
+        Some(owner)
+    } else {
+        None
```

Comment:
(But if the balance is `None` and the authenticated signer is `Some`, it still works, as long as the chain balance is sufficient, right?)

Reply:
Yes, that's right. This is tested in the "and with larger owner account" configurations.